README.md

```diff
@@ -128,14 +128,16 @@ For further information see [the Haskell section](https://nixos.org/nixpkgs/manu
 Build everything:
 
 ```
-nix-build -A exe -o result-exe
+nix-build -A exe --no-out-link
 ```
 
-Run the server:
+Copy the result to a new directory, add configuration, and run!
 
 ```
-cd result-exe
-./backend
+mkdir -p test-app/config/common
+ln -s $(nix-build -A exe --no-out-link)/* test-app/
+cp -r config test-app
```

Comment:
Does this not make the `mkdir` above redundant?

Reply:
It does. Thank you! had several renditions of this and finally decided to just tell them to copy the config in... But forgot to remove stuff